digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/RoadLinkService.scala

```diff
@@ -298,7 +298,7 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
     */
   def getViiteRoadLinksAndChangesFromVVH(bounds: BoundingRectangle, roadNumbers: Seq[(Int, Int)],
                                          municipalities: Set[Int] = Set(), everything: Boolean,
-                                         publicRoads: Boolean): (Seq[RoadLink], Seq[ChangeInfo])= {
+                                         publicRoads: Boolean = false): (Seq[RoadLink], Seq[ChangeInfo])= {

```

Comment:
We're probably better of with code readability if these service level options don't have default values or at least these two either both have a default value or neither does.